src/coreclr/jit/helperexpansion.cpp

```diff
@@ -435,9 +437,9 @@ PhaseStatus Compiler::fgExpandThreadLocalAccess()
 //                             that access fields marked with [ThreadLocal].
 //
 // Arguments:
-//    block - Block containing the helper call to expand
-//    stmt  - Statement containing the helper call
-//    call  - The helper call
+//    pBlock - Block containing the helper call to expand (may be updated in case of block splitting)
```

Comment:
```suggestion
//    pBlock - Block containing the helper call to expand. If expansion is performed, this is updated to the new block that was an outcome of block splitting.
```

Likewise at other places.